guides/common/modules/proc_recovering-corrupted-content-on-a-smart-proxy.adoc

```diff
@@ -0,0 +1,32 @@
+[id="Recovering_Corrupted_Content_on_{smart-proxy-context}_{context}"]
+= Recovering corrupted content on {SmartProxy}
+
+In case of content corruption on a {SmartProxy}, you can recover it by using the `verify-checksum` command in Hammer CLI.
+The `verify-checksum` command repairs content filtered for content views, lifecycle environments, repositories or all content without any filters on the {SmartProxy}.
```

Comment:
Less fluff. Also I'm not sure if "filter" is a proper term here, because I would intuitively expect a filter when retrieving data as compared to updating (?) data as in this case. Academically speaking, I would call it a "selector" :) But prolly best to leave it out altogether.

```suggestion
The `verify-checksum` command can repair content in a content view, lifecycle environment, repository, or all content on {SmartProxy}.
```